### PR TITLE
DOC python 2.7 admin command

### PR DESCRIPTION
**Pull request by beckermr**

### CODE REVIEW

File: src/user/announcements.rst [hunk 1]
```diff
@@ -4,6 +4,15 @@ Announcements
 2020
 ----
 
+:2020-03-21: Python 2.7 Admin Command Available
+
+   - A webservices admin command is now available to add Python 2.7 back to 
+     feedstocks. Put ``@conda-forge-admin add python 2.7`` in the title on an 
+     issue in your feedstock. The admin webservices bot will then issue a PR
+     adding back Python 2.7. Note that this PR will remove other Python builds
+     and any ``win``, ``aarch64``, or ``ppce64le`` builds. If you want to keep
```
Comment (isuruf) [inline src/user/announcements.rst]:
```suggestion
     and any ``win``, ``aarch64``, or ``ppc64le`` builds. If you want to keep 
```

Comment (beckermr) [reply src/user/announcements.rst]: Thanks!

**Comment by beckermr** [comment]
@conda-forge/core this one is ready for review and merging!

**Comment by jakirkham** [comment]
xref: https://github.com/conda-forge/conda-forge-webservices/pull/345

**Pull request merged**